### PR TITLE
Fix for pytest (python3).

### PR DESCRIPTION
**Pull request by myakove**
get_marker() is gone on pytest (py3).

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = py27,py34,pep8
+envlist = py27,py37,pep8
```
Comment (lukas-bednar) [inline tox.ini]: I believe that it would be good to add matrix for pytest3.x & pytest4.x .
So we have compatibility ensured.

**Review by lukas-bednar — APPROVED** [review]
LGTM

**Comment by lukas-bednar** [comment]
Thank you!

**Comment by lukas-bednar** [comment]
@myakove since this repo is just a customization of [eanxgeek/pytest_marker_bugzilla](https://github.com/eanxgeek/pytest_marker_bugzilla). It would be great if we can integrate this improvement there as well. Would you be so kind  to do that ? If you don't have a time I will deal with that ,it is not problem.

**Comment by myakove** [comment]
@lukas-bednar eanxgeek/pytest_marker_bugzilla is outdated and not maintained.
I think the right thing to do it to ask @eanxgeek to get us maintainer permissions or create our own package in pypy from our fork.

What do you think?

**Comment by lukas-bednar** [comment]
@myakove I have maintainer's permissions. And it is not outdated, it just don't have our customizations on top of it.

**Pull request merged**